### PR TITLE
Draft cleanup assets.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Tests pass?   | ✔️
| Fixed issues  | #34

### CODE REVIEW

File: src/View/Event/ViewEvent.php [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
+     * @var string|bool the view file being rendered.
      */
     private $file;
     /**
      * @var array the parameter array passed to the [[View::render()]] method.
      */
     private $parameters;
 
-    public function __construct(string $file, array $parameters = [])
+    public function __construct($file, array $parameters = [])
```
Comment (samdark) [inline src/View/Event/ViewEvent.php]: This change doesn't seem to be related.

File: src/View/View.php [hunk 2]
```diff
@@ -301,9 +302,10 @@ public function renderFile(string $viewFile, array $parameters = [], ?ViewContex
         // TODO: these two match now
         $requestedFile = $viewFile;
 
-        if ($this->theme !== null) {
+        if (!empty($this->theme)) {
```
Comment (samdark) [inline src/View/View.php]: Doesn't seem to be related.

File: src/View/View.php [hunk 3]
```diff
@@ -651,7 +653,8 @@ public function beginCache(string $id, array $properties = []): bool
     {
         $properties['id'] = $id;
         $properties['view'] = $this;
-        $cache = FragmentCache::begin($properties);
+        $cache = FragmentCache::begin();
```
Comment (samdark) [inline src/View/View.php]: Doesn't seem to be related.

File: src/View/Event/ViewEvent.php (hunk 1 above)
Comment (terabytesoftw) [reply src/View/Event/ViewEvent.php]: Phpstan suggests that it can be string or bool, we should stop working with variables of multiple types.

File: src/View/View.php (hunk 2 above)
Comment (terabytesoftw) [reply src/View/View.php]: Phpstan suggests that it will never be null, with empty we validate that it is not null or empty.

File: src/View/View.php (hunk 3 above)
Comment (terabytesoftw) [reply src/View/View.php]: The widgets do not support configuration of array, now we use functions in stack.

File: src/View/Event/ViewEvent.php (hunk 1 above)
Comment (samdark) [reply src/View/Event/ViewEvent.php]: Bool? Why?

Comment (samdark) [reply src/View/Event/ViewEvent.php]: I'll check it.

**Comment by samdark** [comment]
Thanks!
